### PR TITLE
[TEST] some message

### PR DESCRIPTION
**Pull request by piton13**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/129" target="_blank">ReviewNinja

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+asdf
+fdas
```
Comment (valakar) [inline test.txt]: tesdt

File: test.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+asdf
```
Comment (valakar) [inline test.txt:3]: t

File: test.txt (hunk 1 above)
Comment (kucherenko) [inline test.txt]: wtf?

Comment (piton13) [reply test.txt]: Viktor try to save history of comments in his pull-request. And we try it in test-branch.

**Pull request closed without merging**